src/main/io/vtx.c

```diff
@@ -222,7 +222,12 @@ void vtxUpdate(timeUs_t currentTimeUs)
         switch (currentSchedule) {
         case VTX_PARAM_POWER:
             vtxUpdatePending = vtxProcessPower(vtxDevice);
-            break;
```

Comment:
I would replace such code with a macro since we're in C. @ledvinap what do you think, concealing flow control inside macros is bad?

Reply:
FALLTHROUGH is used in this code.  I think the suggestion is to maybe create a macro for the repeated code?

Reply:
I would simply loop around the `switch` until `vtxUpdatePending` is true.
Only problem is how to break when no packet is ready. Probably best is to latch `currentSchedule` before loop and stop is it is seen again ...

Reply:
it's not always required that an update be required.  Otherwise, the task will be infinitely stuck in the loop

Reply:
store the initial state of `currentSchedule` and break once it matches the initial state?

Reply:
@codecae : #5381